app/code/Magento/InventorySalesAlert/Api/ProductIsSalableInterface.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventorySalesAlert\Api;
+
+use Magento\Catalog\Api\Data\ProductInterface;
+use Magento\InventorySalesApi\Api\Data\SalesChannelInterface;
+
+/**
+ * Is product salable
+ *
+ * Used fully qualified namespaces in annotations for proper work of WebApi request parser
+ *
+ * @api
+ */
+interface ProductIsSalableInterface
```

Comment:
This interface should be placed in current `ProductAlert ` module and should be part of current implementation
`ProductAlert ` module must work without new `InventoryProductAlert `